packages/design-system/src/components/Drawer/DrawerToggle.tsx

```diff
@@ -1,5 +1,5 @@
 import Button, { ButtonProps } from '../Button/Button';
-import React, { useEffect, useRef } from 'react';
+import { useEffect, useRef } from 'react';
```

Comment:
I like that the magic part of React is going away but that we're still explicit about importing the functions that we explicitly use.